core/client/src/main/java/alluxio/client/netty/NettyRemoteBlockReader.java

```diff
@@ -103,10 +106,14 @@ public ByteBuffer readRemoteBlock(InetSocketAddress address, long blockId, long
               .getMessage(response.getType(), RPCMessage.Type.RPC_BLOCK_READ_RESPONSE));
       }
     } catch (Exception e) {
+      Metrics.NETTY_BLOCK_READ_FAILURES.inc();
```

Comment:
Yeah, we should. But the current error type classification (and error handling) is a mess from my point of view. I have opened a Jira to fix these stuff. https://alluxio.atlassian.net/browse/ALLUXIO-2235

I also added a bunch of todos on the server side. When we have a clean classfication of the errors, we can add more metrics. Otherwise it is not meaningful